tests/fastify-plugins/maps.js

```diff
@@ -47,14 +59,43 @@ test('prefix opt is handled correctly', async (t) => {
   }
 })
 
-// TODO: Add similar tests/fixtures for proxied online style and offline fallback style
 test('/style.json resolves style.json of local "default" static map when available', async (t) => {
   const server = setup(t)
 
   server.register(StaticMapsPlugin, {
     prefix: 'static',
     staticRootDir: MAP_FIXTURES_PATH,
   })
+  server.register(OfflineFallbackMapPlugin, {
+    prefix: 'fallback',
+    styleJsonPath: path.join(MAPEO_FALLBACK_MAP_PATH, 'style.json'),
+    sourcesDir: path.join(MAPEO_FALLBACK_MAP_PATH, 'dist'),
+  })
+  server.register(MapServerPlugin)
+
+  await server.listen()
+
```

Comment:
Nit: I didn't test this but I don't think this listen is necessary.

```suggestion
```

Reply:
it's needed because the response needs to resolve the server address to include as part of the body